### DISCUSSION TITLE
Mix Light and Dark Mode in Default Theme?

### DISCUSSION THREAD

**Discussion by itjf-de** [requarks/wiki#4842] [Help / Questions]
Hi there - first of all thanks for this awesome Wiki!

Is there an easy way of mixing the light and dark mode in the default Theme?

I want it to look something like in this screenshot when i use the chrome add-in "Dark Reader"
![image](https://user-images.githubusercontent.com/82982998/147685485-a51185a8-f966-472e-9032-d25690df3383.png)
(the top and navigation like the default light theme and the "v-main__wrap" class in dark mode)

Already tried it with code injection but i don't get a clean result - i guess i'm too much of a noob...

thanks&greetings

**Comment by NGPixel** [comment]
It's possible via code injection but not easily.

It will be possible via the UI in the upcoming 3.x release.
